narwhals/_spark_like/expr.py

```
@@ -557,6 +557,17 @@ def func(
 
         return self._with_window_function(func)
 
+    def fill_null(
+        self,
+        value: Any | None,
+        strategy: Literal["forward", "backward"] | None,
+        limit: int | None,
```

sorry hadn't spotted that we're now silently ignoring these

if we want this to be part of tomorrow's release, can we raise if `strategy` is not `None`?

longer-term solution would be to change the metadata kind in `Expr.fill_null` to `WINDOW` if `strategy` is passed, and preserve it otherwise

---

yea no problem - would you want the same for `limit`?

```python
if strategy is not None: # or (limit is not None)
    msg = "Support for strategies is not yet implemented."
    raise NotImplementedError(msg)
```